test/test_trixi2vtk.jl

```diff
@@ -5,6 +5,21 @@ using Trixi2Vtk
 using ReadVTK
 
 
+# Get the version of Trixi.jl we are testing since the output file name
+# format changed in v0.8.0
+import Pkg
+using UUIDs: UUID
+const LEADING_ZEROS = let
+  trixi_uuid = UUID("a7f1ee26-1774-49b1-8366-f1abc58fbfcb")
+  trixi_version = Pkg.dependencies()[trixi_uuid].version
+  if trixi_version > v"0.7.999999999"
+    "000"
+  else
+    ""
+  end
```

Comment:
Out of curiosity, why not
```suggestion
  if trixi_version >= v"0.8"
    "000"
  else
    ""
  end
```

Reply:
OK :-)